packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -586,6 +586,11 @@ module.exports = (
             const avgWPM = 265
             const wordCount = _.words(pureText).length
             timeToRead = Math.round(wordCount / avgWPM)
+            const avgCJKPM = 400
+            const CJKCount = pureText.match(
```

Comment:
can you elaborate a bit on what we're matching here? I'm sorry but I'm not familiar with CJK can't we combine 2 reading counters into 1?

Reply:
> can you elaborate a bit on what we're matching here? I'm sorry but I'm not familiar with CJK can't we combine 2 reading counters into 1?

Chinese and Japanese words aren't separated by space so the most easy solution is counting characters. What is counted here is Chinese characters(`\p{sc=Han}`) and Japanese characters(`\p{sc=Katakana}\p{sc=Hiragana}\p{sc=Han}`).

It's possible to combine into one pass but that requires not using lodash.words. I will look into that later